snuba/clusters/cluster.py

```diff
@@ -56,6 +56,7 @@ class ClickhouseClientSettings(Enum):
             "replication_alter_partitions_sync": 2,
             "mutations_sync": 2,
             "database_atomic_wait_for_drop_and_detach_synchronously": 1,
+            "allow_suspicious_primary_key": 1,
```

Comment:
@onkar @mcannizz this setting is to get around the problems in the EAP tables that run into the order by issue due to https://github.com/ClickHouse/ClickHouse/pull/61399, I think we discussed before that would should actually fix this migration instead of allowing this setting, but this does seem to work incase we need it for temporary backwards compatibility.

Reply:
@onkar @mcannizz https://github.com/getsentry/snuba/pull/6602 should fix this actually - once that's merged I'll update this PR and I think we should be good to go for snuba CI on 24.3. Next would be testing out sentry CI